tests/integration/customizations/s3/__init__.py

```diff
@@ -25,9 +26,16 @@ def make_s3_files(session, key1='text1.txt', key2='text2.txt'):
     """
     region = 'us-west-2'
     bucket = create_bucket(session)
-    string1 = "This is a test."
-    string2 = "This is another test."
-    client = session.create_client('s3', region_name=region)
+
+    if size:
+        string1 = "*" * size
+        string2 = string1
+    else:
+        string1 = "This is a test."
+        string2 = "This is another test."
+
+    config = Config(s3={"addressing_style": "path"})
```

Comment:
Can you discuss why this is needed?  Your description mentioned you were having an issue with new buckets and large files, but I want to make sure we know *exactly* what was causing the test issues before switching over to path style.